inc/sainexthop.h

```diff
@@ -45,11 +45,64 @@ typedef enum _sai_next_hop_type_t
     SAI_NEXT_HOP_TYPE_MPLS,
 
     /** Tunnel next hop */
-    SAI_NEXT_HOP_TYPE_TUNNEL_ENCAP
+    SAI_NEXT_HOP_TYPE_TUNNEL_ENCAP,
+
+    /** IPv6 Segment Route SID List */
+    SAI_NEXT_HOP_TYPE_SEGMENTROUTE_SIDLIST,
+
+    /** IPv6 Segment Route Endpoint Function */
+    SAI_NEXT_HOP_TYPE_SEGMENTROUTE_ENDPOINT
 
 } sai_next_hop_type_t;
 
 /**
+ * @brief Enum defining Endpoint Action types
```

Comment:
Ah ok, i thought you were planning to support only a subset of the endpoint functions, so no issues here.

Btw, you don't mention anything about the transit functions though. Are you planning to add them at a later time?